### ISSUE TITLE
`@sapui5/ts-types` - broken links in docs

### ISSUE THREAD

**Issue by DetachHead** [SAP/ui5-typescript#303]
```ts
  /**
   * @deprecated (since 1.85) - use {@link module:sap/base/security/URLListValidator.Entry} instead.
   *
   * Entry object of the URLWhitelist.
   */
```
**webstorm:**
![image](https://user-images.githubusercontent.com/57028336/143189908-3080f024-f0fc-4ab7-895d-85de960eb014.png)

**vscode:**
![image](https://user-images.githubusercontent.com/57028336/143190465-9773f59a-598c-4f3d-a3dd-2c24640f0c4a.png)
clicking it takes you here:
![image](https://user-images.githubusercontent.com/57028336/143190529-b4793442-6a91-4ceb-bf0b-d262037fc546.png)

**Comment by akudev** [comment]
TL;DR: not sure whether `module:...` JSDoc links are supposed to work at all right now.

https://github.com/microsoft/TypeScript/issues/16498 was about @link with http(s) links, but [mentions that symbol links are harder to support](https://github.com/microsoft/TypeScript/issues/16498#issuecomment-581703177) due to lacking stuff in the TS compiler APIs.

https://github.com/microsoft/TypeScript/issues/35524 is about the rest and was closed by the "80% solution" in https://github.com/microsoft/TypeScript/pull/41877 which was then reverted and re-tried in https://github.com/microsoft/TypeScript/pull/43312

I frankly don't know whether this PR includes support for module:... links, which were earlier [found to only be used in 2% of cases](https://github.com/microsoft/TypeScript/issues/35524#issuecomment-686698291)

Also related on VSCode side: https://github.com/microsoft/vscode/pull/99922 was closed, waiting for TS side
